src/platform/forms-system/src/js/definitions/address.js

```diff
@@ -146,9 +149,9 @@ export function uiSchema(
   }
 
   const addressChangeSelector = createSelector(
-    ({ formData, path }) => _.get(path.concat('country'), formData),
-    ({ formData, path }) => _.get(path.concat('city'), formData),
-    _.get('addressSchema'),
+    ({ formData, path }) => get(path.concat('country'), formData),
+    ({ formData, path }) => get(path.concat('city'), formData),
```

Comment:
Would this work?

```suggestion
    ({ formData, path }) => formData[path.concat('country')],
    ({ formData, path }) => formData[path.concat('city')],
```

Reply:
That should work, especially since both of those are returning strings, so the object wouldn't get mutated. I ran unit tests locally and everything checks out ✅.